.github/workflows/test.yml

```diff
@@ -26,10 +26,8 @@ jobs:
             display-os: Windows
           - feature-args: ''
```

Comment:
Similar to `lint.yml`, left these in as I figured it's easier to leave around for future feature flags